### DISCUSSION TITLE
How to embed in a react application?

### DISCUSSION THREAD

**Discussion by Sultan-Mohammad** [GMOD/jbrowse-components#1637] [Q&A] [answered]
Hi, how can I embed JBrowser2 in my react application? I don't see any documentation for that, is it even possible? I need to integrate JBrowser into my react application.

**Comment by garrettjstevens** [comment, marked as answer]
There are a couple options depending on what you need. The full JBrowse 2 Web app is built as a single-page application, but if you need it to be inside another page instead of as a separate page, the best option is to use an [`<iframe>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe).

The linear genome view functionality of JBrowse 2 is also published separately as a React component which you can use in your application: https://www.npmjs.com/package/@jbrowse/react-linear-genome-view.
